### PR TITLE
Update modules Dockerfiles

### PR DESCRIPTION
**Pull request by netsandbox**
Update the modules/Dockerfile:

* use apt-get instead of apt in modules/Dockerfile
  Do not use apt as it is meant to be an end-user tool.

* use "-o pipefail" for RUN commands in modules/Dockerfile
  see:
  https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#using-pipes
  https://github.com/hadolint/hadolint/wiki/DL4006

### CODE REVIEW

**Comment by thresheek** [comment]
Thanks for the contribution @netsandbox !

Any chance you would provide the same fix for pipefail for an alpine-based modules Dockerfile?

**Comment by netsandbox** [comment]
@thresheek I have updated now also the alpine Dockerfile.

File: modules/Dockerfile [hunk 1]
```diff
@@ -3,17 +3,17 @@ FROM ${NGINX_FROM_IMAGE} as builder
 
 ARG ENABLED_MODULES
 
-RUN set -ex \
-    && if [ "$ENABLED_MODULES" = "" ]; then \
+SHELL ["/bin/bash", "-exo", "pipefail", "-c"]
```
Comment (thresheek) [inline modules/Dockerfile:6]: Wouldnt this affect all further invocations of a resulting image?  Maybe we should just do it like we did with "set -ex", and add pipefail there for the scripts we explicitely chosen to have it?

Comment (netsandbox) [reply modules/Dockerfile:6]: What negative effect are you thinking of?
In general you don't want to have a image build where a pipe silently fails.
And if you add it only on demand, then you have to remember this when you maybe add later a pipe command.

Many people use this in Dockerfiles and shell scripts for years (including me), without experiencing any problem with this.
It is also the recommended by hadolint: https://github.com/hadolint/hadolint/wiki/DL4006

**Comment by netsandbox** [comment]
@thresheek anything else I can do to get this merged?

**Comment by thresheek** [comment]
Thanks @netsandbox !

**Pull request merged**